apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1579,12 +1579,24 @@ public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> res
     return bundle;
   }
 
+  /**
+   * @param beneIds the {@link Collection} beneficiary IDs to log {@link
+   */
   public static void logBeneIdToMdc(Collection<String> beneIds) {
     if (!beneIds.isEmpty()) {
       MDC.put("bene_id", String.join(", ", beneIds));
     }
   }
 
+  /**
+   * @param beneId the {@link Long} beneficiary ID to log {@link
+   */
+  public static void logBeneIdToMdc(Long beneId) {
+    if (beneId != null) {
+      MDC.put("bene_id", String.valueOf(beneId));
+    }
+  }
+
```

Comment:
Can we just convert the original method to take Longs and keep it at a single method?

Reply:
I would not be surprised if the JVM optimizes these to the same thing. Introducing code duplication to save on a single allocation per call when there are any thousands of allocations in HAPI seems like premature optimization to me.

Reply:
I think we'll convert to an array and varargs signature in the upcoming change anyway so doesn't matter much as long as it works